test-configs.yaml

```diff
@@ -161,9 +161,12 @@ test_plans:
         panfrost_submit
 
   kselftest:
-    rootfs: buildroot_kselftest_ramdisk
+    rootfs: debian_buster_nfs
+    pattern: 'kselftest/{category}-{method}-{protocol}-nfs-kselftest-template.jinja2'
```

Comment:
Ah yes that pattern thing should be fixed in the code.  When both a ramdisk and nfs file systems are provided, nfs should take precedence.  It's OK to work around the issue with a special `pattern` here for now.

Reply:
Fixed in #487.